docs: added a code example with linking and tagging (#143)
fingerprintjs/fingerprintjs-pro-react: README.md

See the full code example in the [examples folder](https://github.com/fingerprintjs/fingerprintjs-pro-react/tree/main/examples/create-react-app).

## Linking and tagging information
The visitorId provided by Fingerprint Identification is especially useful when combined with information you already know about your users, for example, account IDs, order IDs, etc. To learn more about various applications of the linkedId and tags, see [Linking and tagging information](https://dev.fingerprint.com/docs/tagging-information).

> Associate your data with a visitorId using the linkedId or tag parameter of the options that are passed into the `useVisitorData()` hook:

Reviewer: ```suggestion
Associate your data with a visitor ID using the `linkedId` or `tag` parameter of the options object passed into `useVisitorData()`:
```

```jsx
// ...

function App() {
  const {
    isLoading,
    error,
    getData
  } = useVisitorData({
    linkedId: "user_1234",
    tag: {
      userAction: "login",
      analyticsId: "UA-5555-1111-1"
    }
  });

> 

Reviewer: ```suggestion
  // getData({ linkedId: "user_1234" }); 
```
